### ISSUE TITLE
Crashes on start macbook pro M1 Max

### ISSUE THREAD

**Issue by segator** [rancher-sandbox/rancher-desktop#2462]
Error Starting Kubernetes
Error: 
```
/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
```
Last command run:
```
/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl start --tty=false 0
```
Context:
Starting virtual machine

Some recent logfile lines:
```
    at ChildProcess.emit (node:events:390:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12) {
  command: [
    '/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl',
    'start',
    '--tty=false',
    '0'
  ]
}
```

If I run this command on a terminal.

```
"/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl" start --tty=false 0
INFO[0000] Creating an instance "0" from template://default (Not from template://0)
WARN[0000] This form is deprecated. Use `limactl start --name=0 template://default` instead
FATA[0000] open /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/share/lima/examples/default.yaml: no such file or directory
```

**Comment by amandaSheng00** [comment]
I am having the same issue. I am using Mac Pro with M1. Rancher desktop version is 1.4.1.

**Comment by 467892746** [comment]
I am having the same issue. I am using Mac Pro with M1. Rancher desktop version is 1.4.1.

**Comment by jandubois** [comment]
Please provide full log files; there is not enough information in the single error message you have shown.

**Comment by segator** [comment]
How Can I extract full logs?

**Comment by danieldonoghue** [comment]
I also note that the error dialog says it's an x86 build instead of aarch64/arm64 and rancher-desktop was considerably slower using the 1.4.1 that older releases.

**Comment by segator** [comment]
lima.ha.stderr.log
```
{"level":"warning","msg":"This version of QEMU might not be able to boot recent Linux guests on M1 macOS hosts.Reinstall QEMU with the following commits (included in QEMU 7.0.0):\n- https://github.com/qemu/qemu/commit/ad99f64f \"hvf: arm: Use macros for sysreg shift/masking\"\n- https://github.com/qemu/qemu/commit/7f6c295c \"hvf: arm: Handle unknown ID registers as RES0\"\nSee https://github.com/Homebrew/homebrew-core/pull/96743 for the further information.","time":"2022-07-06T12:51:17+02:00"}
{"level":"warning","msg":"field `firmware.legacyBIOS` is not supported for architecture \"aarch64\", ignoring","time":"2022-07-06T12:51:17+02:00"}
{"level":"debug","msg":"firmware candidates = [/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/share/qemu/edk2-aarch64-code.fd /usr/share/AAVMF/AAVMF_CODE.fd /usr/share/qemu-efi-aarch64/QEMU_EFI.fd]","time":"2022-07-06T12:51:17+02:00"}
{"level":"debug","msg":"OpenSSH version 8.6.1 detected","time":"2022-07-06T12:51:17+02:00"}
{"level":"debug","msg":"AES accelerator seems available, prioritizing aes128-gcm@openssh.com and aes256-gcm@openssh.com","time":"2022-07-06T12:51:17+02:00"}
{"level":"info","msg":"Starting QEMU (hint: to watch the boot progress, see \"/Users/aymerici/Library/Application Support/rancher-desktop/lima/0/serial.log\")","time":"2022-07-06T12:51:17+02:00"}
{"level":"debug","msg":"qCmd.Args: [/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/qemu-system-aarch64 -m 20480 -cpu host -machine virt,accel=hvf,highmem=off -smp 9,sockets=1,cores=9,threads=1 -drive if=pflash,format=raw,readonly=on,file=/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/share/qemu/edk2-aarch64-code.fd -boot order=d,splash-time=0,menu=on -drive file=/Users/aymerici/Library/Application Support/rancher-desktop/lima/0/basedisk,media=cdrom,readonly=on -drive file=/Users/aymerici/Library/Application Support/rancher-desktop/lima/0/diffdisk,if=virtio -cdrom /Users/aymerici/Library/Application Support/rancher-desktop/lima/0/cidata.iso -netdev user,id=net0,net=192.168.5.0/24,dhcpstart=192.168.5.15,hostfwd=tcp:127.0.0.1:56489-:22 -device virtio-net-pci,netdev=net0,mac=52:55:55:1e:74:42 -netdev vde,id=net1,sock=/private/var/run/rancher-desktop-shared.ctl -device virtio-net-pci,netdev=net1,mac=52:55:55:f6:96:60 -netdev vde,id=net2,sock=/private/var/run/rancher-desktop-bridged_en6.ctl -device virtio-net-pci,netdev=net2,mac=52:55:55:f9:55:68 -device virtio-rng-pci -display none -vga none -device ramfb -device qemu-xhci,id=usb-bus -device usb-kbd,bus=usb-bus.0 -device usb-mouse,bus=usb-bus.0 -parallel none -chardev socket,id=char-serial,path=/Users/aymerici/Library/Application Support/rancher-desktop/lima/0/serial.sock,server=on,wait=off,logfile=/Users/aymerici/Library/Application Support/rancher-desktop/lima/0/serial.log -serial chardev:char-serial -chardev socket,id=char-qmp,path=/Users/aymerici/Library/Application Support/rancher-desktop/lima/0/qmp.sock,server=on,wait=off -qmp chardev:char-qmp -name lima-0 -pidfile /Users/aymerici/Library/Application Support/rancher-desktop/lima/0/qemu.pid]","time":"2022-07-06T12:51:17+02:00"}
{"level":"info","msg":"Waiting for the essential requirement 1 of 5: \"ssh\"","time":"2022-07-06T12:51:17+02:00"}
{"level":"debug","msg":"executing script \"ssh\"","time":"2022-07-06T12:51:17+02:00"}
{"level":"debug","msg":"executing ssh for script \"ssh\": /usr/bin/ssh [ssh -F /dev/null -o IdentityFile=\"/Users/aymerici/Library/Application Support/rancher-desktop/lima/_config/user\" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o NoHostAuthenticationForLocalhost=yes -o GSSAPIAuthentication=no -o PreferredAuthentications=publickey -o Compression=no -o BatchMode=yes -o IdentitiesOnly=yes -o Ciphers=\"^aes128-gcm@openssh.com,aes256-gcm@openssh.com\" -o User=aymerici -o ControlMaster=auto -o ControlPath=\"/Users/aymerici/Library/Application Support/rancher-desktop/lima/0/ssh.sock\" -o ControlPersist=5m -p 56489 127.0.0.1 -- /bin/bash]","time":"2022-07-06T12:51:17+02:00"}
{"level":"debug","msg":"qemu[stderr]: qemu-system-aarch64: Number of SMP CPUs requested (9) exceeds max CPUs supported by machine 'mach-virt' (8)","time":"2022-07-06T12:51:17+02:00"}
{"level":"debug","msg":"stdout=\"\", stderr=\"kex_exchange_identification: read: Connection reset by peer\\r\\nConnection reset by 127.0.0.1 port 56489\\r\\n\", err=failed to execute script \"ssh\": stdout=\"\", stderr=\"kex_exchange_identification: read: Connection reset by peer\\r\\nConnection reset by 127.0.0.1 port 56489\\r\\n\": exit status 255","time":"2022-07-06T12:51:17+02:00"}
{"error":"exit status 1","level":"info","msg":"QEMU has exited","time":"2022-07-06T12:51:17+02:00"}
```

**Comment by jandubois** [comment]
The error message spells it out:

> qemu-system-aarch64: Number of SMP CPUs requested (9) exceeds max CPUs supported by machine 'mach-virt' (8)"

This makes it a duplicate of #2190
